com.unity.formats.alembic/Documentation~/reference.md

```diff
@@ -2,9 +2,15 @@
 
 This section provides details on the properties for the following windows or components:
 
-- [Alembic Exporter component](ref_Exporter.md), which allows you to export GameObjects, Cameras, and Points (in the Particle System).
-- [Alembic Point Cloud component](ref_PointCloud.md), which allows you to customize the shape and behavior of the point cloud.
-- [Alembic Point Renderer component](ref_PointRenderer.md), which allows you to customize the appearance of the particles in the point cloud.
-- [Alembic Shot Asset component](ref_Shot.md), which allows you to where where the Alembic Shot clip stops playback.
-- [Alembic Stream Player component](ref_StreamPlayer.md), which allows you to play back animation from the Alembic file in either Timeline or Unity Animation.
-- [Import Settings for Alembic](ref_Importer.md), which allows you to customize importing the data from the Alembic file.
+| Element                                                        |                                                                | Description                                                                                                                                          |
```

Comment:
I'm probably missing something obvious here 😅 but what is the difference between the items listed in the first columns vs. the second column? (and if there is a reason, could we add a header for the second column?)

Reply:
I was trying to do something like [this file](https://github.com/Unity-Technologies/com.unity.formats.alembic/blob/main/com.unity.formats.alembic/Documentation~/import-options-cameras.md) to show that the components are related 😆 but I can remove the second level if it's clearer!

Reply:
Gotcha! I think it makes a bit more sense in that other page because it shows a specific relationship, the left column is the property name and the right column are the possible choices for that property.

I can see the relationship you are representing in this table but I'm not totally sure it's improving the user's understand of how these components are linked. I honestly don't have that strong of an opinion on this 😛  . Either way is totally fine but I would personally lean more towards a single column.

Technically, you can also have an `Alembic Stream Player` without the importer so the relationship is not always there (ex: add a Stream Player component on a gameobject and you can point to an alembic file outside of the project)

Reply:
Ahhhh that makes sense! I'll put them back in two columns then :) However, should I place components in the same order as the table of content or in alphabetical order?

Reply:
Hmm, good question. Your order makes sense logically but since it's a more "dry" reference page maybe alphabetically is the standard?

@MaryPerchanok do you have an opinion on this? Do you think we should order the components alphabetically or logically (components that are similar or have similar purpose stay together)

Reply:
My two cents here: consider the reference section like a dictionary, so its TOC structure should be in alphabetical order. Other perspectives that are more contextual/conceptual (similar purpose or complementary roles) would be addressed in specific topics of the doc.